app/views/posts/index.html.erb

```diff
@@ -1 +1,12 @@
-<h1>Here is a list of posts for a given user</h1>
+<%= render "shared/user_box", user: @user %>
+<% @posts.each do |post| %>
+  <a class = "post-box" href="<%= user_post_path(@user, post) %>">
+    <div class = "bio-box-container">
+      <%= render "shared/post_box", post: post, user: @user%>
+    </div>
+      <%= render "shared/comment_box", post: post, limit: 5 %>
```

Comment:
- [ ] Kindly check comment above :point_up:  and make use of `#recent_comments` to retrieve the 5 most recent comments of a post.

Reply:
Thank you for your suggestion, It's Done!

Reply:
This change has been done well